src/benchmarks/micro/libraries/System.Numerics.Tensors/Perf_NumberTensorPrimitives.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+using System.Linq;
+
+namespace System.Numerics.Tensors.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD, Categories.JIT)]
+    [GenericTypeArguments(typeof(int))]
+    [GenericTypeArguments(typeof(float))]
+    [GenericTypeArguments(typeof(double))]
+    public class Perf_NumberTensorPrimitives<T>
+        where T : INumber<T>, IBitwiseOperators<T, T, T>
+    {
+        [Params(128, 6 * 512 + 7)]
+        public int BufferLength;
+
+        private T[] _source1;
+        private T[] _source2;
+        private T[] _source3;
+        private T _scalar1;
+        private T[] _ones;
+        private T[] _destination;
+
+        [GlobalSetup]
+        public void Init()
+        {
+            _source1 = ValuesGenerator.Array<T>(BufferLength, seed: 42);
+            _source2 = ValuesGenerator.Array<T>(BufferLength, seed: 43);
+            _source3 = ValuesGenerator.Array<T>(BufferLength, seed: 44);
+            _scalar1 = ValuesGenerator.Value<T>(seed: 45);
+            _ones = Enumerable.Repeat(T.One, BufferLength).ToArray();
+            _destination = new T[BufferLength];
+        }
+
+        #region Unary Operations
+        [Benchmark]
+        public void Abs() => TensorPrimitives.Abs<T>(_source1, _destination);
+
+        [Benchmark]
+        public void Negate() => TensorPrimitives.Abs<T>(_source1, _destination);
```

Comment:
may be TensorPrimitives.Negate?

Reply:
Why that operator specifically?

Reply:
Oh, I see, the existing test is testing the wrong method.